### PR TITLE
feat: create component edit mode

### PR DESCRIPTION
**Pull request by DobrovskayaElena**
Just a simple version. I am working on it.

### CODE REVIEW

**Review by Vsevolod1020 — APPROVED** [review]

File: src/app/modules/questions-block/questions-loading.service.ts [hunk 1]
```diff
@@ -35,7 +36,35 @@ export class QuestionsLoadingService {
...
+  updateQuestion(data: any, id: string) {
+    return this.http.put<any>('http://localhost:3000/posts' + id, data).pipe(
+      map((res: any) => {
+        return res;
+      })
+    );
+  }
+
+  deleteQuestion(id: string) {
+    return this.http.delete<any>('http://localhost:3000/posts' + id).pipe(
```
Comment (team1Js21) [inline src/app/modules/questions-block/questions-loading.service.ts]: please move `http://localhost:3000` to `environment.ts` file

File: src/app/pages/questions-edit/questions-table/questions-table.component.ts [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  }
+
+  postQuestionDetails() {
+    this.questionModel.id = this.formValue.value.id;
+    this.questionModel.text = this.formValue.value.text;
+    this.questionModel.type = this.formValue.value.type;
+
+    this.questionsLoadingService.postQuestion(this.questionModel).subscribe(
+      (res: any) => {
+        console.log(res);
```
Comment (team1Js21) [inline src/app/pages/questions-edit/questions-table/questions-table.component.ts]: please remove `console.log`

File: src/app/pages/questions-edit/questions-table/questions-table.component.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        let ref = document.getElementById('cancel');
```
Comment (team1Js21) [inline src/app/pages/questions-edit/questions-table/questions-table.component.ts]: Always use `const` if the variable should not change

File: src/app/modules/questions-block/questions-loading.service.ts [hunk 4]
```diff
@@ -35,7 +36,35 @@ export class QuestionsLoadingService {
     );
   }
 
-  audioTryCount() {
-    
+  postQuestion(data: any) {
```
Comment (team1Js21) [inline src/app/modules/questions-block/questions-loading.service.ts]: Please specify the type for the `data`

**Review by Ars6300 — APPROVED** [review]

**Pull request merged**